### PR TITLE
Add get-follower-ids-api

### LINKED ISSUE #340: Support get follower IDs api
Doc: https://developers.line.biz/ja/reference/messaging-api/#get-follower-ids

### PR DESCRIPTION
**Pull request by hinoguma**
Fixes #340

### CODE REVIEW

File: src/LINEBot.php [hunk 1]
```diff
@@ -95,6 +95,51 @@ public function getProfile($userId)
...
 
+    /**
+     * Gets the list of User IDs of users who have added your LINE Official Account as a friend.
+     * These users' IDs won't be included in the obtained list of user IDs:
+     * ・ Users who blocked the target LINE Official Account after adding it as a friend.
+     * ・ Users who haven't consented to their profile information being obtained.
+     *
+     * This feature is only available for LINE@ Approved accounts or official accounts.
+     *
+     * @param string $start continuationToken
```
Comment (moririnson) [inline src/LINEBot.php:106]: string|null

File: src/LINEBot.php [hunk 2]
```diff
@@ -95,6 +95,51 @@ public function getProfile($userId)
...
+     * @return Response
+     */
+    public function getFollowerIds($start = null)
+    {
+        $url = sprintf('%s/v2/bot/followers/ids', $this->endpointBase);
```
Comment (moririnson) [inline src/LINEBot.php:111]: Could you make url without `sprintf` ?
Like this.
https://github.com/line/line-bot-sdk-php/pull/342/files#diff-22797c186358a12c4ade20a7d7891165463f37cfdf3bb0905ad33893051ac5b5R172

I want to use `sprintf` when is needed to concat 3 or more strings.

File: tests/LINEBot/GetFollowerIdsTest.php [hunk 3]
```diff
@@ -0,0 +1,73 @@
+<?php
+
+/**
+ * Copyright 2016 LINE Corporation
```
Comment (moririnson) [inline tests/LINEBot/GetFollowerIdsTest.php:4]: 2020

**Review by moririnson — CHANGES_REQUESTED** [review]

File: tests/LINEBot/GetFollowerIdsTest.php (hunk 3 above)
Comment (hinoguma) [reply tests/LINEBot/GetFollowerIdsTest.php]: @moririnson 
thanks for reviews.
fixed.

File: src/LINEBot.php [hunk 4]
```diff
@@ -95,6 +95,50 @@ public function getProfile($userId)
         return $this->httpClient->get($this->endpointBase . '/v2/bot/profile/' . urlencode($userId));
     }
 
+    /**
+     * Gets the list of User IDs of users who have added your LINE Official Account as a friend.
+     * These users' IDs won't be included in the obtained list of user IDs:
+     * ・ Users who blocked the target LINE Official Account after adding it as a friend.
+     * ・ Users who haven't consented to their profile information being obtained.
```
Comment (Vaduz) [inline src/LINEBot.php:102]: Please remove unicode characters
```suggestion
     * - Users who blocked the target LINE Official Account after adding it as a friend.
     * - Users who haven't consented to their profile information being obtained.
```
Or remove them completely.

File: src/LINEBot.php [hunk 5]
```diff
@@ -330,7 +374,7 @@ public function getRoomMemberProfile($roomId, $userId)
      * This feature is only available for LINE@ Approved accounts or official accounts.
      *
      * @param string $groupId Identifier of the group
-     * @param string $start continuationToken
+     * @param string|null $start continuationToken
```
Comment (moririnson) [inline src/LINEBot.php:377]: thanks!

File: src/LINEBot.php [hunk 6]
```diff
@@ -95,6 +95,50 @@ public function getProfile($userId)
...
+    public function getFollowerIds($start = null)
+    {
+        $params = is_null($start) ? [] : ['start' => $start];
+        return $this->httpClient->get($this->endpointBase . '/v2/bot/followers/ids', $params);
+    }
+
+    /**
+     * Gets the list of User IDs of users who have added your LINE Official Account as a friend.
+     * These users' IDs won't be included in the obtained list of user IDs:
+     * ・ Users who blocked the target LINE Official Account after adding it as a friend.
```
[the remark this answers is not in the document]
Comment (hinoguma) [reply src/LINEBot.php]: Oops, thanks.fixed

**Review by moririnson — APPROVED** [review]

**Review by Vaduz — APPROVED** [review]

**Pull request merged**